jsonnet/prometheus-operator/prometheus-operator.libsonnet

```diff
@@ -2,6 +2,7 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
 
 {
   _config+:: {
+    labels: { 'apps.kubernetes.io/name': 'prometheus-operator' },
```

Comment:
this should be nested in a `prometheusOperator` object

Reply:
👍